features/step_definitions/admin_steps.rb

```diff
@@ -1,13 +1,47 @@
 require 'cucumber/rspec/doubles'
 
-Given /I am an admin user/ do
-  visit root_url
+# Prereq: In order for admin capability, "testadminuser@berkeley.edu" must exist within ENV["TEST_ADMIN_EMAILS"]
+# Returns a OAuth2 token associated with email "testadminuser@berkeley.edu"
+Given /I have an admin email/ do
+  OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new({
+    provider: 'google_oauth2',
+    uid: '123545',
+    info: {
+      name: 'Admin User',
+      first_name: "Admin",
+      last_name: "User",
+      email: "testadminuser@berkeley.edu",
+      school: "UC Berkeley",
+    },
+    credentials: {
+      token: 'test_token'
+    }
+  })
 end
 
+# Returns a OAuth2 token associated with email "randomemail@gmail.com"
+Given /I have a random email/ do
```

Comment:
We should either make this truly random using `Faker` or replace the word "random" with something a little more communicative, non-admin?